test/JDBC/input/BABEL_COL_NAME-vu-prepare.sql

```diff
@@ -0,0 +1,58 @@
+CREATE DATABASE babel_3172_test_db;
+GO
+
+USE babel_3172_test_db;
+GO
+
+CREATE TABLE sys_column_name_vu_t_column_name(
+    id int,
+    names char
+)
+GO
```

Comment:
Can you please create a table in another schema and write few tests around that?